src/components/config/parse.ts

```diff
@@ -24,13 +24,56 @@
 
 import { Config } from '../../interfaces/config'
 import { readFileSync } from 'fs'
+import { RequestConfig } from '../../interfaces/request'
+import { Probe } from '../../interfaces/probe'
 
-export const parseConfig = (configPath: string): Config => {
+export const parseHarFile = (fileContents: string): Config => {
```

Comment:
as mentioned by @nicnocquee here https://github.com/hyperjumptech/monika/pull/286#discussion_r673710410

it should be moved to their own file

Reply:
done